constants/csv_parser.ts

```diff
@@ -0,0 +1,224 @@
+import { Question, Answer } from '../models'
+import fs from 'fs'
+import path from 'path'
+import { parse } from 'csv-parse/sync'
+
+// The type containing columns that we care about in the Flowchart CSV
+type CsvType = {
+  Id: string
+  Name: string
+  'Line Source': string
+  'Line Destination': string
+  'Text Area 1': string
+  'Contained By': string
+}
```

Comment:
hmm, my first thought was if these string fields could be converted into enums? i'd imagine this was prob considered before. if possible, i think having the fields like `Line Source` could be more readable since it seems like they're used often throughout here

i did quick internet investigation and found [this ](https://stackoverflow.com/questions/55142177/how-to-build-a-type-from-enum-values-in-typescript)
basically, would it be possible to convert the `type CsvType` to smth like this with [`Record`](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type)?
```suggestion
enum CsvKeys {
  ID: 'Id',
  NAME: 'Name',
  LINE_SOURCE: 'Line Source'
  ....
}

type CsvType = Record<CsvKeys, string>;
```

Reply:
Wait this page also has so many other useful type things that I didn't know TypeScript had, TY VERA!